scripts/buildCSS.js

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const CleanCSS = require('clean-css');
+const compileCSS = require('react-with-styles-interface-css-compiler');
+const registerMaxSpecificity = require('react-with-styles-interface-css/dist/utils/registerMaxSpecificity').default;
+const registerCSSInterfaceWithDefaultTheme = require('./utils/registerCSSInterfaceWithDefaultTheme').default;
+
+const args = process.argv.slice(2);
+const optimizeForProduction = args.includes('-o') || args.includes('--optimize');
```

Comment:
not critical, but it might be cleaner to update this to use `yargs` or something.